### DISCUSSION TITLE
Run in background

### DISCUSSION THREAD

**Discussion by udobehm** [Devolutions/UniGetUI#2104] [Q&A]
Hi, does WingetUI have to run all the time in the background in order to update my apps once a week (if set in the options to check for updates once a week). I would like WingetUI only to run when it checks for the updates. Is it possible to configure WingetUI that way?

Thanks

**Comment by marticliment** [comment]
Please be aware that the week starts counting once WingetUI is launched, so the timer is reset each time your computer is restarted

**Comment by udobehm** [comment]
Means I can disable the WingetUI autostart so there is no symbol in the taskbar but WingetUI still will be run for checking for updates once a week?

**Reply by marticliment** [reply]
No. You must have WinGetUI running in order for the timer to work

**Comment by udobehm** [comment]
Ok. Think you might add an option so that I don't have the program to be run all the time but set the Windows scheduler for example that it runs the app and checks for updates once a week? Could you add the option that your app automatically adds this scheduler entry without the need to do this manually?
I'd prefer that your app would not run all the time in the background for saving CPU and memory...

**Comment by marticliment** [comment]
Will think about it

**Reply by udobehm** [reply]
Ok, hope you may implement that 👍
